docs/HACKING.CRYPTO

```diff
@@ -362,19 +362,19 @@ Converts the absolute value of bn into big-endian form and store it at
 val. val must point to _libssh2_bn_bytes(bn) bytes of memory.
 Returns the length of the big-endian number.
 
-void _libssh2_bn_rand(_libssh2_bn *bn, int bits, int top, int bottom);
+int _libssh2_bn_rand(_libssh2_bn *bn, int bits, int top, int bottom);
 Generates a cryptographically strong pseudo-random number of bits in
 length and stores it in bn. If top is -1, the most significant bit of the
 random number can be zero. If top is 0, it is set to 1, and if top is 1, the
 two most significant bits of the number will be set to 1, so that the product
 of two such random numbers will always have 2*bits length. If bottom is true,
-the number will be odd.
+the number will be odd. Returns 1 on success, 0 otherwise.
```

Comment:
I don't think it is at the moment, because at least most backend related functions use 0 on success and -1 on error. In the WinCNG backend there is not a single function that uses 1 on success and 0 on error.